shell/platform/windows/flutter_windows_engine_unittests.cc

```diff
@@ -255,5 +256,35 @@ TEST(FlutterWindowsEngine, DispatchSemanticsAction) {
   EXPECT_TRUE(called);
 }
 
+TEST(FlutterWindowsEngine, AddPluginRegistrarDestructionCallback) {
+  std::unique_ptr<FlutterWindowsEngine> engine = GetTestEngine();
+  EngineModifier modifier(engine.get());
+
+  MockEmbedderApiForKeyboard(modifier,
+                             std::make_shared<MockKeyResponseController>());
```

Comment:
Why does this test need a mock keyboard handler?

Reply:
This is required by `engine.RunWithEntrypoint`.